docs/components/BlockieAvatar.md

```diff
@@ -0,0 +1,33 @@
+---
+sidebar_position: 8
+---
+
+# BlockieAvatar
+
+Show a blockie (bar code profile icon) component for a given public address.
+
+If the address has an ENS avatar linked to it, this avatar will be displayed. Otherwise, a generated blockie will be shown.
```

Comment:
Actually this is not true 😅, Its just a dumb component and only shows blockie corresponding to that address

the `ensImage` prop must be passed by user and this can be any image URL (not at all related to ens) and if this prop is passed the component will show this image instead of blockie

Reply:
Don't know from where I took it or why I thought it worked like that! Sorry 😅

I changed it for this sentence: "The autogenerated blockie can be manually replaced by another image that we pass through the `ensImage` prop.". But can delete it if it's enough talking about that in the Prop table